gradle.properties

```diff
@@ -2,12 +2,12 @@ name=trinkets
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-minecraft_version=1.16.2
-yarn_mappings=1.16.2+build.19
-loader_version=0.9.1+build.205
+minecraft_version=1.16.4
```

Comment:
Is there a reason we're building against 1.16.4 instead of one of the snapshot versions since we're targeting 1.17? Not that I think it really matters for our purposes, but I'm just curious.

Reply:
A number of people have said they'd want to use 3.0.0 on 1.16.4, so I was planning to do the upgrade last if at all, I'm thinking of potentially changing the plan we originally had and converting all mods that use trinkets to 3.0.0 manually and pr that stuff in, since 7 months is a long wait